ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.19.yaml

```diff
@@ -277,6 +277,23 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest
       OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:initial
     workflow: openshift-upgrade-aws
+- always_run: false
+  as: e2e-aws-ovn-ipsec-upgrade
```

Comment:
can you flip these to aws-ovn-upgrade-ipsec for consistency, most upgrade jobs with a modifier like this use that format. same for ipsec-serial